hbase-shell/src/main/ruby/shell/commands/processlist.rb

```diff
@@ -27,10 +27,10 @@ def help
 
   hbase> processlist
   hbase> processlist 'all'
-  hbase> processlist 'general'
-  hbase> processlist 'handler'
-  hbase> processlist 'rpc'
-  hbase> processlist 'operation'
+  hbase> processlist 'general'     //non-RPC Tasks, such as compact, flush etc
```

Comment:
Thanks for this @bsglz! Documentation like this is a little inconsistent in format throughout our commands, but I think it would be good to follow the precedent of the RSGroup commands and place these comments on the preceding lines with a ruby-style comment (`#`) since the shell is ruby.

Ex: https://github.com/apache/hbase/blob/8003a15835690de6d36a91cef3d51eea3ba60129/hbase-shell/src/main/ruby/shell/commands/set_peer_tableCFs.rb#L35

Reply:
Will fix later.
Thanks.